cmd/goacme/main_test.go

```diff
@@ -22,15 +22,15 @@ func TestDiscoAliasFlag(t *testing.T) {
 		{defaultDisco, []string{"-d", "https://disco"}, "https://disco"},
 	}
 	for i, test := range tests {
-		var a discoAliasFlag = test.a
+		var daf = test.a
```

Comment:
fair, I guess for me I read the use of the type there as a way to provide a description for what `a` actually means. By removing this it becomes less clear what this variable represents, which is why I moved it to `daf`